src/main/java/teammates/ui/webapi/GetFeedbackSessionsAction.java

```diff
@@ -88,13 +97,38 @@ public JsonResult execute() {
                 } else {
                     feedbackSessionAttributes = logic.getFeedbackSessionsListForInstructor(instructors);
                 }
+                if (instructors.isEmpty()) {
+                    throw new EntityNotFoundException(NO_SUCH_USER_FOUND);
+                }
+                String emailAddress = instructors.get(0).getEmail();
+                feedbackSessionAttributes = feedbackSessionAttributes.stream()
+                        .map(instructorSession -> instructorSession.sanitizeForInstructor(emailAddress))
+                        .collect(Collectors.toList());
```

Comment:
I brought this up in my initial conversation yesterday, this change while good to have breaks the frontend and will require more complex logic than this.

Reply:
Oh wait, I forgot, is it because of the tooltip/displayed status? If I'm not wrong, the intended tooltip/displayed status is supposed to be (in short) closed with ongoing extensions, not open with ongoing extensions, right? In that case, I don't think I can change the submission status for instructors at all here. I think in this case, for things like making the Submit button enabled in the instructor home page, it must be handled in the front-end. Unless I'm missing something, I do not think this can be handled from the back-end. The alternative would be for me to keep this and Fergus to change his tooltip/displayed status logic, but I think it's easier if I just remove this?

So now adjusting the submission status and filtering deadlines will only occur for students. For instructors, the logic will remain the same in `GetFeedbackSessionsAction`.

Reply:
I think we should remove for instructors for now, since we are already slightly behind schedule. This is the most straightforward way at the moment. I plan to see if it's possible to refactor this part to be more consistent after the initial release and to make sure only instructors with proper permissions can view the extensions page.